### ISSUE TITLE
REQ 15c should be clarified.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/apa#206]
"REQ 15c: Provide an XR angel or helper for the user with a cognitive or learning disability."

It isn't clear what this "angel" or "helper" is or would do in the context of the User Need. Would it be a real person, or a virtual persona, or either? Would it perform tasks requiring timely responses of the user, and if so, how would this work (e.g., automatically in some ases, or at the user's instruction)?

I don't think we need an extensive description, but a sentence in REQ 15c explaining what this is about and how it might work to satisfy the User Need would greatly help here. At the moment, it's all left to the reader's imagination to speculate about what is intended.

**Comment by RealJoshue108** [comment]
@jasonjgw Agreed. I'm not clear on this either, and how the help manifests is largely an implementation issue IMO.
There are various references to help in the COGA content usable doc, for example in [related user stories ](https://www.w3.org/TR/coga-usable/#help-user-story)and [patterns](
https://www.w3.org/TR/coga-usable/#objective-7-provide-help-and-support):

**Comment by RealJoshue108** [comment]
I'm not sure about this requirement, and wonder if it should be remove - either that or would it be sufficient to remove the reference to 'XR angels' and provide a ref to the COGA usable doc?

REQ 15c: Provide  help for users with a cognitive or learning disability. 

with links to content usable?

**Comment by jasonjgw** [comment]
I think it would be better to give clearer indications of what "help" consists in. It could be anything from explanatory information or textual annotations in an application to human assistance in real time, and I'm not sure what forms it should take in an XR/immersive context.

**Comment by RealJoshue108** [comment]
@jasonjgw I've added a note to say:

>The term 'help' for REQ 15c may vary from explanatory information, textual/symbolic annotations in an application to human assistance in real time.
